### ISSUE TITLE
Some dynamic variables in pre-scripts are not resolving in newman

### ISSUE THREAD

**Issue by dpihlaja** [postmanlabs/newman#2340]
I swear I'm not crazy, but it seems that _some_ dynamic variables are not resolving in `newman`, however they resolve in the Postman app, and the Collection Runner. 

This could be because of how I am using the pre-request script and stringifying an object, but to me, if this works in the app, it should work in `newman`

I've attached a collection with a console log, all you have to do to repro is run it. 
https://www.getpostman.com/collections/baded51de7112dc47f9b

From the app:
```
{"foo":[{"guid":"f459a1fa-4a92-45bf-ab71-c28a05e38ec4","randomInt":"985","randomPrice":"192.23"}]}
```

From `newman`:
```
{"foo":[{"guid":"cb18503c-f96f-4248-b806-d60b9f96c06
  │ 3","randomInt":"313","randomPrice":"{{$randomPrice}}"}]}
```

I'm not seeing any distinction between $randomInt and $randomPrice in the docs.
https://learning.postman.com/docs/postman/variables-and-environments/variables-list/

I haven't gone through every dynamic variable, but I imagine if this one is an edge case, there would be more. 

 1. Newman Version (can be found via `newman -v`): 4.4.1
 2. OS details (type, version, and architecture): MacOS 10.14.6
 3. Are you using Newman as a library, or via the CLI? CLI
 3. Did you encounter this recently, or has this bug always been there: First time using DV's
 4. Expected behaviour: Dynamic Variables should resolve as they do in Postman, and the Collection Runner
 5. Command / script used to run Newman: `newman run $collection.json`

https://www.getpostman.com/collections/baded51de7112dc47f9b

Steps to reproduce the problem:

1. Run the Postman collection attached in the Collection runner, and in `newman`
2. Observe the console.log statement, and compare the `randomPrice` value
3. The value is not being resolved in newman, but it is in the runner and app

**Comment by codenirvana** [comment]
@dpihlaja This is because you are using an older version of Newman which lacks support for a few dynamic variables.
I hope updating Newman will fix this issue.

**Comment by dpihlaja** [comment]
I updated newman, but didn't restart my shell so I thought I was on current :(

Confirmed fix in 5.0.0. Thanks! I'm going to go find some sand to bury my head in now. 

Thanks again for the reply, unblocks me on a project.
